src/AuthReg.cc

```diff
@@ -30,18 +31,18 @@
 void
 Auth::Init()
 {
-    debugs(29,DBG_IMPORTANT,"Startup: Initializing Authentication Schemes ...");
+    debugs(29, Important(68), "Startup: Initializing Authentication Schemes ...");
```

Comment:
If you are OK with downgrading this message (currently ID 68), please let me know, and I will gladly downgrade it to level 2 as well:

```suggestion
    debugs(29, 2, "Startup: Initializing Authentication Schemes ...");
```